app-modules/alert/src/Models/Alert.php

```diff
@@ -144,6 +146,11 @@ public static function executeFromCampaignAction(CampaignAction $action): bool|s
         // Do we need to be able to relate campaigns/actions to the RESULT of their actions?
     }
 
+    public function user()
```

Comment:
Let's change this to `createdBy` and make sure it references the column one the name has changed.

Reply:
Changed